### DISCUSSION TITLE
Slash Cooldowns

### DISCUSSION THREAD

**Discussion by searchify7** [nextcord/nextcord#715] [General]
It's more better to add a  cooldown for application commands as built-in without making our own one.
It will be more nice if devs add it for slash commands, since the cooldown for legacy commands is a little bit different for application command backend.

**Comment by ghost** [comment]
Yes, please.

**Comment by Skelmis** [comment]
In the meantime you can use [this](https://cs.skelmis.co.nz/?q=nextcord_slash_cooldowns) to add cooldowns to your slash commands. Its a happy medium while they don't exist in the repo itself.

**Reply by searchify7** [reply]
I already know that there is a custom cooldown that can work for slash, but it will be more poggers if there was a specific one for slash.

**Comment by mno9** [comment]
hey, does anyone know how to add cooldown (user id) in slash commands? 
with an error like if cooldown hasnt passed it will give an error
